flink-sql-runner/pom.xml

```diff
@@ -121,4 +128,38 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+        <id>coverage</id>
```

Comment:
Also will need `-Pcoverage` for the `mvn` command in `.github/worksflows/integration.yaml`